controllers/db/admin.settings.controller.ts

```diff
@@ -0,0 +1,57 @@
+import sequelize from "../../src/db/db";
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { AdminSettings } from '../../src/db/models/AdminSettings'
+import models from '../../src/db/sequelize'
+import { Op, Sequelize } from "sequelize";
+
+export const listAdminSettings = async (req: NextApiRequest, res: NextApiResponse) => {
+    const adminSettings = {};
+    try {
+        const adminSettings = await models.AdminSettings.findAll({
```

Comment:
looks like you settings adminSettings above on line 8. consts are only defined onc